internal/handlers/common/params.go

```diff
@@ -213,3 +215,43 @@ func parseTypeCode(alias string) (typeCode, error) {
 
 	return code, nil
 }
+
+func addNumbers(v1, v2 any) (any, error) {
+	switch v1 := v1.(type) {
+	case float64:
+		switch v2 := v2.(type) {
+		case float64:
+			return v1 + v2, nil
```

Comment:
That code would happily sum NaN and infinity, but there are no tests for that

Reply:
I've added a test for that case.